### ISSUE TITLE
Support OCCURS UNBOUNDED for Cobol v6

### ISSUE THREAD

**Issue by smedilol** [TypeCobolTeam/TypeCobol#1305]
**Is your feature request related to a problem? Please describe.**
**CHILD OF #616**
Support new keyword `UNBOUNDED` in syntax of `OCCURS` for Cobol v6

TypeCobol code that you'd like to write.
```cobol
       01 Group1.
         05 Var1 pic 9(18).
         05 MyArray occurs 1 to unbounded depending on Var1.
```

See full syntax for `OCCURS`:
![image](https://user-images.githubusercontent.com/13764482/59589156-97cbb400-90e9-11e9-8daf-e9595a8c5d28.png)

Out of scope:
The following rules will not be checked. We'll rely on the IBM compiler to check them:
## Unbounded table
- A table with an OCCURS clause that specifies UNBOUNDED.
- You can reference unbounded tables in COBOL syntax anywhere a
table can be referenced.

## Unbounded group
- A group that contains at least one unbounded table.
- You can define unbounded groups only in the LINKAGE
SECTION. Either alphanumeric groups or national groups can be
unbounded.
- You can reference unbounded groups in COBOL syntax anywhere
an alphanumeric or national group can be referenced, with the
following exceptions:
   - You cannot specify unbounded groups as a BY CONTENT argument
in a CALL statement.
   - You cannot specify unbounded groups as data-name-2 on the
PROCEDURE DIVISION RETURNING phrase.
   - You cannot specify unbounded groups as arguments to intrinsic
functions, except as an argument to the LENGTH intrinsic function.
The total size of an unbounded group at run time must be less
than 999,999,999 bytes.
 - For unbounded tables and groups, the effect of the SSRANGE
compiler option is limited. For more information, see SSRANGE in
the Enterprise COBOL Programming Guide.

**Comment by fm-117** [comment]
No further development is required as the full syntax of OCCURS clause is already supported.
Besides, OCCURS UNBOUNDED is not part of v6.1, it was introduced in v5.1.
